### ISSUE TITLE
How to fix "Error listen tcp6 [::1]:45125: cannot assign requested address" without disabling ipv6

### ISSUE THREAD

**Issue by SiriusRed** [kubernetes/helm#2658]
I have problem, like in #1410, #2223, #2482

```sh
$ helm ls
E0708 19:22:42.516032     120 portforward.go:212] Unable to create listener: Error listen tcp6 [::1]:45125: bind: cannot assign requested address
```

I run helm in container, where I can't disable ipv6 and change /etc/hosts

What another way to fix it?

**Comment by technosophos** [comment]
You might be able to change `localhost` to `127.0.0.1` in cmd/helm/helm.go:205 and rebuild Helm. That will hard-code Helm to use the IPv4 address instead of whatever is found in /etc/hosts.

**Comment by hobti01** [comment]
Why is this reported as an `ERROR` if it does not impact functionality?  The exit code is still `0` even though this output occurs on stderr - which is not intuitive.
```
# helm status some-release; echo $?
E0727 07:23:34.907857     108 portforward.go:212] Unable to create listener: Error listen tcp6 [::1]:42721: bind: cannot assign requested address
LAST DEPLOYED: Wed Jul 26 17:56:59 2017
NAMESPACE: default
STATUS: DEPLOYED

RESOURCES:
==> v1beta1/Deployment
NAME                                       DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
...

0
```

**Comment by debben** [comment]
I'm having the same problem right now. I'm trying to write a Helm plugin for Visual Studio Team Services. They have shared build agents and I can't turn on IPV6. They let you stand up your own agents, and I was able to stand up docker image of the vsts-agent with the flag "--sysctl net.ipv6.conf.all.disable_ipv6=0". For many users of VSTS though, they may not have their own hosted agents. It would be nice if the localhost was changed to an explicit 127.0.0.1 [like was done with the local repository url](https://github.com/kubernetes/helm/pull/1427)
